src/lua/help_en_US.lua

```diff
@@ -264,24 +264,24 @@ Obviously it will work, so just type <tutorial("next")!> now.
 ]];
 
 [[
-Tutorial -- Screen #10 -- Packages
-==================================
+Tutorial -- Screen #10 -- Modules
+=================================
 
-Many developers have gone to the trouble of making
-packages of functions (sometimes called "modules")
-that have a general utility.
+Many developers have gone to the trouble of making modules,
+i.e. packages of functions that have a general utility.
+In the Lua world, modules are called "rocks".
 More in the Luarocks list: http://luarocks.org/
 
-Most packages have to be "required", with the syntax
-variable_name = require("package-name")
+Most modules have to be "required", with the syntax
+variable_name = require("module-name")
 which should look familiar because earlier you said
 console = require("console")
 
 At this point, if you just say the variable_name,
-you’ll see a list of the package’s members and
+you’ll see a list of the module’s members and
 functions. If then you use a "." operator as in
-package-variable-name.function_name()
-you’ll invoke a package’s function.
+module-variable-name.function_name()
```

Comment:
I guess 'module-variable-name' should be 'variable_name' (as it is written above). Moreover, a dash is not valid part of a variable name in Lua.